### DISCUSSION TITLE
Need help with a bale wrapping state

### DISCUSSION THREAD

**Discussion by Kettunen78** [jason0611/FS22_DashboardLive#77] [General]
How would I get a buzzer and indicator light or a text that the bale is ready to be dropped from the wrapper? For example Claas Rollant 455 Uniwrap
p.s. a camera screen would be a nice addition.

**Comment by jason0611** [comment]
I have to find out, if there is a corresponding value available in the game. Then this feature could be added.
A camera screen seems to be impossible, because there isn't a usable shader available for this feature.

**Reply by Kettunen78** [reply]
Thank you for your reply. I'll wait and see if you can find the things you need. I wondered if it would be possible to do with a mirror. I've been trying to see how it could be done.

**Comment by jason0611** [comment]
Maybe it's solvable without modification of DBL. It could be possible to use DBL's fillLevel-function:
`<dashboard valueType="base" cmd="fillLevel" trailer="1" partition="1" cond="moreequal" factor="100" condValue="100" option="percent" joints="..." ... />`

**Reply by Kettunen78** [reply]
I'll try this

**Comment by Kettunen78** [comment]
No luck with this one. I think "fillLevel" doesn't read the correct position because the wrapper doesn't use "fillunit". The wrapper is apparently based on an animation that when completed will trigger a drop if I understand correctly.
